### DISCUSSION TITLE
How to set up self-monitoring for Nightingale

### DISCUSSION THREAD

**Discussion by wy281704862** [ccfos/nightingale#1166] [General]
Open-Falcon有提供状态接口，夜莺好像没有看到。如果有麻烦告知一下。

**Comment by UlricQin** [comment]
n9e-webapi n9e-server 这两个模块都提供符合prometheus协议的`/metrics`接口，可以单独搭建一个简单的prometheus来利用这俩接口做监控

**Comment by wy281704862** [comment]
好的 谢谢
